joomla/joomla-cms
# [4.0] Fix Article Versions JS error

Pull Request for Issue #23811 .

### Summary of Changes

Added special function that will be used when browser is IE11 and returns the closest element

### Testing Instructions

npm ci is required
In IE11, open any article in your Article Manager
Click the "Versions" button
Check any of the checkboxes

### Actual result

No error in console now

### Documentation Changes Required

None
